src/components/Form/DomainConfiguration.tsx

```diff
@@ -0,0 +1,144 @@
+'use client';
+import React from 'react';
+import { useState } from 'react';
+import { getSubdomain } from '../../modules/MultiDomain/domains';
+import { useDomainStatus } from '../../modules/MultiDomain/UseDomainStatus';
+import { AlertCircle, CheckCircle, XCircle } from 'lucide-react';
+
+export const InlineSnippet = ({ children }: { children: string }) => {
+  return (
+    <span className='inline-block rounded-md bg-blue-100 px-1 py-0.5 font-mono text-blue-900 dark:bg-blue-900 dark:text-blue-100'>
```

Comment:
for styling, to apply globally: 
- for now we don't handle light/dark mode, we are dark by default, so you can remove dark: everywhere, and duplicate props
- don't override font, here font-mono, this is defined globally

Reply:
tbh, i just copied and pasted it out of some vercel code base. I can remove that. or you help me overhaul everything.